RNTester/js/RNTesterApp.ios.js

```diff
@@ -249,7 +250,12 @@ const styles = StyleSheet.create({
     fontSize: 19,
     fontWeight: '600',
     textAlign: 'center',
-    color: DynamicColorIOS({light: 'black', dark: 'white'}), // TODO(OSS Candidate ISS#2710739)
+    color:
+      // [TODO(macOS ISS#2323203)
+      Platform.OS === 'macos'
+        ? DynamicColorMacOS({light: 'black', dark: 'white'})
+        : DynamicColorIOS({light: 'black', dark: 'white'}), // TODO(OSS Candidate ISS#2710739)
+    // ]TODO(macOS ISS#2323203)
```

Comment:
nit spacing is off

Reply:
This is done by prettier, which doesn't know these comments have any relation. I guess I could add a comment to disable prettier, but that feels a bit much 🤷

Reply:
(FWIW I feel your pain)